java/core/src/java/org/apache/orc/impl/ColumnStatisticsImpl.java

```diff
@@ -179,10 +365,10 @@ public int hashCode() {
     private boolean hasMinimum = false;
     private boolean overflow = false;
 
-    IntegerStatisticsImpl() {
+    IntegerColumnStatisticsImpl() {
```

Comment:
Let's not change the class name.

Reply:
Sorry, this was aftereffect of refactoring. Fixed it.